File: benchmark/results/python-multilang/run-002/b-baseline/output/metaengine_demo/shipping/value_objects/shipment_route.py
"""ShipmentRoute value object for the shipping domain."""

from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class ShipmentRoute:
    """Immutable route a shipment travels along."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str

File: benchmark/results/ts-monolith-opus-script/run-002/b-baseline/output/src/orders/fulfillment/value-objects/ShipmentRoute.ts
import { Id } from "../../../shared/value-objects/Id";
import { Timestamp } from "../../../shared/value-objects/Timestamp";

/** Named route taken by a shipment. */
export interface ShipmentRoute {
  id: Id;
  createdAt: Timestamp;
  updatedAt: Timestamp;
  name: string;
  description: string;
}
